packages/admin/blocks-admin/src/blocks/types.tsx

```diff
@@ -75,6 +75,7 @@ export interface BlockMethods<
     childBlockCount?: (state: State) => number;
     previewContent: (state: State, context?: BlockContext) => PreviewContent[];
     dynamicDisplayName?: (state: State) => React.ReactNode;
+    getAnchors?: (state: State) => string[];
```

Comment:
the `get` prefix is strange here, we also don't have getDynamicDisplayName, getPreviewContent etc.